src/main/scala/mesosphere/marathon/core/externalvolume/impl/providers/DVDIProvider.scala

```diff
@@ -68,18 +67,19 @@ private[impl] case object DVDIProvider extends ExternalVolumeProvider {
       // containerizer. the docker containerizer simply ignores them.
       applyOptions(volBuilder, dockerVolumeParameters(volume))
 
+      val mode = if (mount.readOnly) MesosVolume.Mode.RO else MesosVolume.Mode.RW
```

Comment:
This conversion happens a few times in code. May be make it a `def` in `object VolumeMount`?

Reply:
Done.